axelor-production/src/main/java/com/axelor/apps/production/service/manuforder/ManufOrderServiceImpl.java

```diff
@@ -942,6 +945,13 @@ public void merge(List<Long> ids) throws AxelorException {
       if (manufOrder.getNote() != null && !manufOrder.getNote().equals("")) {
         note += manufOrder.getManufOrderSeq() + " : " + manufOrder.getNote() + "\n";
       }
+      
+      if (Strings.isNullOrEmpty(manufOrder.getMoCommentFromSaleOrder())) {
```

Comment:
I think this is the wrong condition, we concatenate the comment if the comment is **not** null or empty, the current code will throw a NPE.